test/fixtures/condition/condition-unless.htl

```diff
@@ -0,0 +1 @@
+<sly data-sly-test="${ foo == 'false' }">Foo</sly>
```

Comment:
Shouldn't it be `false` as a boolean, currently you're comparing it to a string?